custom_components/unifiprotect/unifi_protect_server.py

```diff
@@ -40,293 +53,248 @@ def __init__(
         self.device_data = {}
 
         urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
-        self.req = requests.session()
-        self._api_auth_bearer_token = self._get_api_auth_bearer_token()
```

Comment:
I had to move these 2 calls out of __init__ as you can do anyc calls in the an `__init__` method.  The api_auth_bearer_token will get set the first time HA calls the `update` method on this class.